app/controllers/api/vcs.php

```diff
@@ -0,0 +1,1019 @@
+        }
+
+        // Create / Update installation
+        if (!empty($installationId)) {
+            $privateKey = App::getEnv('VCS_GITHUB_PRIVATE_KEY');
+            $githubAppId = App::getEnv('VCS_GITHUB_APP_ID');
+            $github->initialiseVariables($installationId, $privateKey, $githubAppId);
+            $owner = $github->getOwnerName($installationId);
+
+            $projectInternalId = $project->getInternalId();
+
+            $vcsInstallation = $dbForConsole->findOne('vcsInstallations', [
+                Query::equal('installationId', [$installationId]),
+                Query::equal('projectInternalId', [$projectInternalId])
+            ]);
+
+            if ($vcsInstallation === false || $vcsInstallation->isEmpty()) {
+                $teamId = $project->getAttribute('teamId', '');
+
+                $vcsInstallation = new Document([
+                    '$id' => ID::unique(),
+                    '$permissions' => [
+                        Permission::read(Role::team(ID::custom($teamId))),
+                        Permission::update(Role::team(ID::custom($teamId), 'owner')),
+                        Permission::update(Role::team(ID::custom($teamId), 'developer')),
+                        Permission::delete(Role::team(ID::custom($teamId), 'owner')),
+                        Permission::delete(Role::team(ID::custom($teamId), 'developer')),
+                    ],
+                    'installationId' => $installationId,
+                    'projectId' => $projectId,
+                    'projectInternalId' => $projectInternalId,
+                    'provider' => 'github',
+                    'organization' => $owner,
+                    'personal' => $personalSlug === $owner
+                ]);
+
+                $vcsInstallation = $dbForConsole->createDocument('vcsInstallations', $vcsInstallation);
+            } else {
+                $vcsInstallation = $vcsInstallation->setAttribute('organization', $owner);
+                $vcsInstallation = $vcsInstallation->setAttribute('personal', $personalSlug === $owner);
+                $vcsInstallation = $dbForConsole->updateDocument('vcsInstallations', $vcsInstallation->getId(), $vcsInstallation);
+            }
+        } else {
+            throw new Exception(Exception::GENERAL_ARGUMENT_INVALID, 'Installation of the Appwrite GitHub App on organization accounts is restricted to organization owners. As a member of the organization, you do not have the necessary permissions to install this GitHub App. Please contact the organization owner to create the installation from the Appwrite console.');
+        }
+
+        $response
+            ->addHeader('Cache-Control', 'no-store, no-cache, must-revalidate, max-age=0')
+            ->addHeader('Pragma', 'no-cache')
+            ->redirect($redirect);
+    });
+
+App::get('/v1/vcs/github/installations/:installationId/repositories')
+    ->desc('List repositories')
```

Comment:
```suggestion
    ->desc('List Repositories')
```

We need to fix all the description. Today all services follow title case format.